data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,211 @@ mission "Hai Pilgrimage"
+			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
+			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
+			`	"But-" Eeeya objects.`
+			choice
+				`	"Anoo is right; the Unfettered are a noble people. I would recommend joining them."`
+					goto joinuhai
+				`	"There's no good reason to join the Unfettered. What have they ever done except kill innocent Hai?"`
+					goto "don't join"
+			label joinuhai
+			`	"Well... I really don't know..."`
+			choice
+				`	"Never mind, you're right. Joining the Unfettered is a bad idea after all."`
+				`	"I stand by their guns. The Unfettered are honorable."`
+					goto reallyjoin
+			label "don't join"
+			`	"I'm sorry, Anoo, but I really think Captain <first> is right. Fundamentally, I'm not the kind of person to be friendly with a group who starves and yet paints an exotic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are precious enough, let alone cookies. Hospital work here can only pay so much.`
+			`	"Captain, could you take me back to <planet> please? I'd like to sign the forms and give you some stock in my Acorn Delights factory. In the meantime, here is <payment> for what you've done so far."`
+			choice
+				`	"I'd be glad to take you back."`
+				`	"I'll take you, but I'll notice that you haven't mentioned what we came so far for."`
+			`	Eeeya smiles at her granddaughter. "Before I leave, I was just reminded about a present I have for you." She leads you and Anoo back to <ship>. "Captain, could you open the cargo hold and take us inside?" Anoo is astonished at the tons of acorn cookies waiting for her inside. "Thank you so much, grandma!"`
+			choice
+				`	"We should get going."`
+			`	Eeeya turns to you and says, "You're right, we should. Thank you so much for reuniting us." She gives Anoo one last hug, and boards your ship.`
+				accept
+
+			label reallyjoin
+			action
+				set "acorn delights: eeeya joined"
+			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the Unfettered. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`
+			choice
+				`	"Sure thing. I'd be glad to take you back."`
```

Comment:
You could add something here about the player being surprised Eeeya accepted joining the Unfettered so easily, which may provide an opportunity for her to explain why she's joining them. Perhaps have her mention here that she has few to no ties left in Hai space, and that's part of the reason she's deciding to join the Unfettered.